config/job_conf.xml

```diff
@@ -233,35 +209,11 @@
       <param id="max_pod_retrials">3</param>
       <param id="docker_enabled">true</param>
     </destination>
-    <destination id="container-mw2isa" runner="k8s">
-      <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
-      <param id="docker_owner_override">phnmnl</param>
-      <param id="docker_image_override">mw2isa</param>
-      <param id="docker_tag_override">v0.5.0_cv0.1.20</param>
-      <param id="max_pod_retrials">1</param>
-      <param id="docker_enabled">true</param>
-    </destination>
-    <destination id="container-isatab2json" runner="k8s">
-      <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
-      <param id="docker_owner_override">phnmnl</param>
-      <param id="docker_image_override">isatab2json</param>
-      <param id="docker_tag_override">v0.8.0_cv0.2.39</param>
-      <param id="max_pod_retrials">3</param>
-      <param id="docker_enabled">true</param>
-    </destination>
-    <destination id="container-json2isatab" runner="k8s">
-      <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
-      <param id="docker_owner_override">phnmnl</param>
-      <param id="docker_image_override">json2isatab</param>
-      <param id="docker_tag_override">v0.8.0_cv0.2.26</param>
-      <param id="max_pod_retrials">3</param>
-      <param id="docker_enabled">true</param>
-    </destination>
     <destination id="container-isatab-create" runner="k8s">
       <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
       <param id="docker_owner_override">phnmnl</param>
       <param id="docker_image_override">isatab-create</param>
-      <param id="docker_tag_override">v0.9.5_cv0.3.6.37</param>
+      <param id="docker_tag_override">latest</param>
```

Comment:
Can we put here a release tag for isatab-create? or is this not going to be used as the dynamic destination is the one used now?

Reply:
There should be a tag present now in the latest version of this PR.